packages/e2e/tests/apps-installation.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test, expect, Page } from "@playwright/test";
+import { logInIntoDashboard } from "./operations/log-in-to-dashboard";
+import { installTheApp } from "./operations/install-app";
+import { appUrls, routing } from "../setup/routing";
+import { AppManifest } from "@saleor/app-sdk/types";
+import { assertAppAvailable } from "./assertions/assert-app-available";
+
+// target every app including staging. This should be config from the outside
```

Comment:
```suggestion
// target every app including staging. This should be config from the outside
```

What do you mean by "the outside"? Can you specify? Is this a todo comment for the future or a note?